lib/burndown_chart.rb

```diff
@@ -195,6 +195,10 @@ def load_sprint(burndown_dir, number = nil)
   def update(options)
     burndown_data_path = load_sprint(options['output'] || Dir.pwd, options[:sprint_number])
 
+    if options.has_key?('board-id')
```

Comment:
why adding this in the hash? You can directly do this in the assignment. Please move this to https://github.com/openSUSE/trollolo/pull/125/files#diff-17ddcbd394dd49848e146577f8d3d845L199

It would be like:

``` Ruby
burndown_data.board_id = options['board-id'] || board_id

Reply:
I did not remember at first, but now it I do: that does not work. I tried it like that first, but it does not update the board-id in the burndown.yaml which lead me to the solution you can see above (which does work)

Reply:
ok... :see_no_evil: 

Then write it at least in one single line following [Ruby Style guide](https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier), so:

``` Ruby
@data['meta']['board_id'] = options['board-id'] if options.has_key?('board-id')
```